console/src/main/webapp/manager/app/components/user/user.es6

```diff
@@ -48,6 +60,11 @@ class UserController {
     this.required = $injector.get('UserRequired').get()
   }
 
+  hasDelegation () {
+    if (!this.delegation) return false
+    return (this.delegation.orgs.length !== 0) || (this.delegation.roles.length !== 0)
```

Comment:
A valid delegation must have at least one roles **and** one org. I think you should use `&&` instead of `||` ?

Reply:
:+1: